sqlx-sqlite/src/lib.rs

```diff
@@ -127,7 +127,21 @@ pub static CREATE_DB_WAL: AtomicBool = AtomicBool::new(true);
 /// UNSTABLE: for use by `sqlite-macros-core` only.
 #[doc(hidden)]
 pub fn describe_blocking(query: &str, database_url: &str) -> Result<Describe<Sqlite>, Error> {
+    #[cfg(feature = "sqlx-toml")]
+    let opts = {
+        let mut opts: SqliteConnectOptions = database_url.parse()?;
+
+        let config = sqlx_core::config::Config::try_from_crate_or_default().unwrap_or_default();
```

Comment:
Suppressing the error here is not ideal. This is only if there's an actual issue parsing the config, which the user should be notified about.

Reply:
I've ended up adding a new "ConfigFile" variant to the main error type. That significantly improves the error reporting story in both of the places where you pointed it out.